src/components/AssetItem/AssetItem.js

```diff
@@ -0,0 +1,178 @@
+// @flow
+
+import React, { PureComponent, Fragment } from 'react'
+
+import classNames from 'classnames'
+import { handle } from 'utils/eventHandlers'
+import AssetBalance from 'components/AssetBalance'
+import { JAssetSymbol, JText, JCard, JIcon, JFlatButton } from 'components/base'
+
+type Props = {
+  deleteAssetItem: () => void,
+  +name: string,
+  +symbol: string,
+  +balance: number,
+  +isCustom: boolean,
+}
+
+type StateProps = {|
+  isChange: boolean,
+  isHoveredEdit: boolean,
+  isHoveredTrash: boolean,
+  isDeleteDialog: boolean,
+|}
+
+class AssetItem extends PureComponent<Props, StateProps> {
+  static defaultProps = {
+    fiatBalance: 0,
+    balance: 0,
+    isCustom: false,
+  }
+
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      isChange: false,
+      isHoveredEdit: false,
+      isHoveredTrash: false,
+      isDeleteDialog: false,
+    }
+  }
+
+  onHoverEdit = (isHoveredEdit: boolean) => {
+    this.setState({ isHoveredEdit })
+  }
+
+  onHoverTrash = (isHoveredTrash: boolean) => {
+    this.setState({ isHoveredTrash })
+  }
+
+  toggleChange = (isChange: boolean) => {
+    this.setState({
+      isChange,
```

Comment:
laconic way:
`toggleChange = () => this.setState({ isChange: !this.state.isChange })`
and usage:
`onClick={this.toggleChange}`